README.md

```diff
@@ -119,31 +45,54 @@ text file `_config/dirs.txt`, one per line. E.g.
     user@eris1n2.partners.org:/data/pnl
 
 * NOTE: replace user with whomever has sudo access.
-    
+
+Modify `html-report-generator/generate_usage_report.py` to change defaults in the html report.
+
 # Running
 
+## Scripts
     logbigfiles <dir> # finds files > 300M in <dir>, writes csv output to _data/logbigfiles/
+
     logdirsizes <dir> # writes directory sizes in `<dir>` (default depth 3) to `_data/logdirsizes/`
+
     logdirsizesall # calls `logdirsizes` for each directory in _config file `_config/dirs.txt`
+
     mydf # prints file system usage for each directory in _config file `_config/dirs.txt`
+
     logdf # writes `mydf` output to `_data/logdf/`
 
-# Output
+## Script Output
 
 Each script generates a csv file when you run it, and saves it to its log directory.
-`logdirsizesall` and `logdf` should be run regularly to get a revealing profile of
-disk usage changes.
+
+`logdf` should be run more frequently (~daily) to get a generalized view of total disk usage trends
+
+`logdirsizesall` should be run less frequently (~weekly) to get a more nuanced view of disk usage of individual directories
 
 # Report
 
 ## 1. Make report
 
-In `report-lib/` there's an R markdown document that generates a disk usage graph and a summary
-of last week's disk space changes. Generate it by running:
+The python file `html-report-generator/generate_usage_report.py` generates an HTML report with detailed diskusage data. Run it like this:
+
+```bash
+conda activate /path/to/diskusage/env
+python html-report-generator/generate_usage_report.py
+```
 
-    makereport.sh
+This saves the report as `_data/htmlreport/report-{filesystem_name}-{date}.html`.
```

Comment:
Put this line after line 95.

Reply:
Delete the rest of lines 78-91.

Reply:
I shortened this entire section, let me know what you think.